jobs/openshift/cluster/integreatly/openshift-cluster-integreatly-test.yaml

```diff
@@ -0,0 +1,27 @@
+---
+- job:
+    name: openshift-cluster-integreatly-test
+    display-name: 'Openshift Cluster Integreatly Test'
+    project-type: pipeline
+    parameters:
+      - string:
+          name: 'installerGitUrl'
```

Comment:
Just noticed this is called installer instead of installation. I think in almost all other job defs we have called this installationGitUrl as it represents the installation repo (https://github.com/integr8ly/installation/). Probably best to keep it consistent.